### ISSUE TITLE
Uncaught TypeError: Cannot read property 'add' of null

### ISSUE THREAD

**Issue by ganeshkbhat** [Gert-dev/php-integrator-navigation#5]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.2.4
**System**: Microsoft Windows 8 Single Language
**Thrown From**: [php-integrator-navigation](https://github.com/Gert-dev/php-integrator-navigation) package, v0.2.4

### Stack Trace

Uncaught TypeError: Cannot read property 'add' of null

```
At /C:/Users/Ganesh/.atom/packages/php-integrator-navigation/lib/AbstractProvider.coffee:167

TypeError: Cannot read property 'add' of null
    at ClassProvider.module.exports.AbstractProvider.registerEvents (file:///C:/Users/Ganesh/.atom/packages/php-integrator-navigation/lib/AbstractProvider.coffee:113:21)
    at ClassProvider.module.exports.ClassProvider.registerEvents (file:///C:/Users/Ganesh/.atom/packages/php-integrator-navigation/lib/ClassProvider.coffee:53:9)
    at ClassProvider.module.exports.AbstractProvider.registerEventsForPane (file:///C:/Users/Ganesh/.atom/packages/php-integrator-navigation/lib/AbstractProvider.coffee:93:18)
    at file:///C:/Users/Ganesh/.atom/packages/php-integrator-navigation/lib/AbstractProvider.coffee:75:18
    at Emitter.module.exports.Emitter.emit (C:\atom\resources\app.asar\node_modules\event-kit\lib\emitter.js:86:11)
    at PaneContainer.module.exports.PaneContainer.didDestroyPane (C:\atom\resources\app.asar\src\pane-container.js:301:27)
    at Pane.module.exports.Pane.destroyed (C:\atom\resources\app.asar\src\pane.js:653:55)
    at Pane.module.exports.Model.destroy (C:\atom\resources\app.asar\src\model.js:31:58)
    at Pane.module.exports.Pane.destroy (C:\atom\resources\app.asar\src\pane.js:635:39)
    at Pane.module.exports.Pane.removeItem (C:\atom\resources\app.asar\src\pane.js:407:21)
    at Pane.module.exports.Pane.destroyItem (C:\atom\resources\app.asar\src\pane.js:449:16)
    at atom-tabs.TabBarView.onClick (C:\atom\resources\app.asar\node_modules\tabs\lib\tab-bar-view.js:695:17)
```

### Commands

```
  2x -0:37.4.0 click (atom-text-editor.editor.is-focused)
     -0:35.4.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:34.5.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:30.2.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.theme-atom-light-syntax.theme-atom-light-ui)
  3x -0:30.1.0 click (div.title.icon.icon-tools)
  3x -0:27 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.theme-atom-light-syntax.theme-atom-light-ui)
     -0:21.5.0 click (div.title)
     -0:20.8.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.theme-atom-light-syntax.theme-atom-light-ui)
  4x -0:20.8.0 click (div.title.icon.icon-tools)
     -0:16.8.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.theme-atom-light-syntax.theme-atom-light-ui)
     -0:15.5.0 click (span.name.icon.icon-file-text)
     -0:15.5.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.theme-atom-light-syntax.theme-atom-light-ui)
  4x -0:15.3.0 click (span.name.icon.icon-file-text)
     -0:06.2.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.theme-atom-light-syntax.theme-atom-light-ui)
     -0:06.2.0 click (div.close-icon)
  2x -0:06.1.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.theme-atom-light-syntax.theme-atom-light-ui)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "atom-light-syntax"
    ],
    "disabledPackages": [
      "php-introspector"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-mocha, v0.0.2
atom-typescript, v7.14.1
auto-encoding, v0.5.0
auto-indent, v0.5.0
auto-reveal-active-file, v0.3.1
autoupdate-packages, v1.3.1
javascript-snippets, v1.0.0
language-smarty-php, v0.5.0
less-autocompile, v1.0.0
linter, v1.11.3
node-debugger, v1.3.0
php-analyser, v0.2.0
php-composer-completion, v0.3.4
php-cs-fixer, v2.5.1
php-debug, v0.1.4
php-extract-method, v0.2.1
php-getters-setters, v0.5.5
php-integrator-annotations, v0.2.2
php-integrator-autocomplete-plus, v0.3.0
php-integrator-base, v0.3.1
php-integrator-navigation, v0.2.4
php-integrator-tooltips, v0.2.1
phpunit, v1.2.0
phpunit-snippets, v0.1.0

# Dev
No dev packages
```

**Comment by Gert-dev** [comment]
Hello. Did you happen to manually install the package? If so, did you run `apm install` in the package directory?

**Comment by ganeshkbhat** [comment]
yes

**Comment by Gert-dev** [comment]
Ok, are you still experiencing this issue and if so, is there any example code base I can reproduce it with?

**Comment by pguimier** [comment]
I just got de same error twice, trying to split window in two panes.

**Comment by Gert-dev** [comment]
Thanks for reporting this. Should be fixed in 0.6.1.
